Added example launchfile

wuehr1999 opened the pull request:
Hi @aentinger, I created a launch file example here.

aentinger reviewed (commented):
Hi :coffee: :wave:

Launch file example looks good, however why's the `openmv` submodule deleted in this PR?

wuehr1999 commented:
Hi  @aentinger thank You for the hint, I added the submodule again :+1:

aentinger reviewed (changes requested):
You've got some mess together here :wink: let me help you out.

aentinger reviewed (approved):
Should work now, but `git submodule update --init --recursive`is very slow at my end. Can you give it a try @wuehr1999 ?

wuehr1999 commented:
@aentinger  the problem with the submodule was that I forgot updating the module path when creating the ROS2 version (for development i just moved the OpenMV repository manually ...).
```git submodule update --init --recursive``` is slow as it also clones the submodules in the submodule, like micropython and HAL drivers. For the ROS node recursive cloning is not necessary, I added documentation in the README.

aentinger commented:
That makes perfect sense! Thanks for cleaning this up :rocket: :+1: :bow: .

wuehr1999 commented:
=> will merge
